app.js

```diff
@@ -0,0 +1,50 @@
+const glob = require("glob");
+const jsonConcat = require("json-concat");
+glob("./server/**/*.json",null,function(er,files){
+    if(er){
+        console.log(er);
+    }
+    else{
+        jsonConcat({
+            src : files,
+            dest: "./server/db.json"
+        }, function (json){
+            console.log(json);

```

Comment:
was this required?

Reply:
removed in the current code as it was not required. Intially I was testing app.js